sibyl/resources/model.py

```diff
@@ -56,7 +56,7 @@ def get(self, model_id):
           400:
             $ref: '#/components/responses/ErrorMessage'
         """
-        model = schema.Model.find_one(id=model_id)
+        model = schema.Model.find_one(model_id=model_id)
```

Comment:
Why does this GET function have `model_id` as a function parameter instead of having it in the request arguments like other functions?

Reply:
All our basic get commands (get entity, model, feature, etc.) have the parameters as function parameters. I believe this allows the parameter to be passed directly in the URL route. I've been meaning to make another PR that goes through all these options more carefully for each API call to make sure we are making the right choice